hopp/simulation/technologies/grid.py

```diff
@@ -125,8 +130,66 @@ def simulate_grid_connection(
             self.schedule_curtailed = np.array([gen - schedule if gen > schedule else 0. for (gen, schedule) in
                                             zip(total_gen, lifetime_schedule)])
             self.schedule_curtailed_percentage = sum(self.schedule_curtailed)/sum(lifetime_schedule)
+
+            # NOTE: This is currently only happening for load following, would be good to make it more general
+            #           i.e. so that this analysis can be used when load following isn't being used (without storage)
+            #           for comparison 
+            N_hybrid = len(self.generation_profile)
+
+            final_power_production = total_gen
+            schedule = [x for x in lifetime_schedule]
+            print(len(final_power_production), len(schedule))
+            hybrid_power = [(final_power_production[x] - (schedule[x]*0.95)) for x in range(len(final_power_production))]
+
+            load_met = len([i for i in hybrid_power if i  >= 0])
+            self.time_load_met = 100 * load_met/N_hybrid
+
+            final_power_array = np.array(final_power_production)
+            power_met = np.where(final_power_array > schedule, schedule, final_power_array)
+            self.capacity_factor_load = np.sum(power_met) / np.sum(schedule) * 100
+
+            print('Percent of time firm power requirement is met: ', np.round(self.time_load_met,2))
+            print('Percent total firm power requirement is satisfied: ', np.round(self.capacity_factor_load,2))
```

Comment:
print statements

Reply:
Really these should be part of the logger, we have this problem in a number of places in the codebase. I'm not sure the logger is working as intended right now, but I'll add them as logs anyway